### PR TITLE
adding cost calculations to connect and move_to

### PR DESCRIPTION
**Pull request by tylerjw**
cost calculations are based on sum of trajectory duration

### CODE REVIEW

File: core/src/stages/connect.cpp [hunk 1]
```diff
@@ -219,9 +219,16 @@ SubTrajectoryPtr Connect::merge(const std::vector<robot_trajectory::RobotTraject
                                 const std::vector<planning_scene::PlanningSceneConstPtr>& intermediate_scenes,
                                 const moveit::core::RobotState& state)
 {
+	// set cost
+	double cost = 0;
+	for (const auto& trajectory: sub_trajectories) {
+		for (const double& distance: trajectory->getWayPointDurations())
```
Comment (rhaschke) [inline core/src/stages/connect.cpp]:
```suggestion
		cost += trajectory->getWayPointDurationFromStart();
```

File: core/src/stages/move_to.cpp [hunk 2]
```diff
@@ -234,11 +234,18 @@ bool MoveTo::compute(const InterfaceState &state, planning_scene::PlanningSceneP
 		success = planner_->plan(state.scene(), *link, target_eigen, jmg, timeout, robot_trajectory, path_constraints);
 	}
 
+	// set cost
+	double cost = 0;
+	for (const double& distance: robot_trajectory->getWayPointDurations()) {
```
Comment (rhaschke) [inline core/src/stages/move_to.cpp]: see above

File: core/src/stages/connect.cpp (hunk 1 above)
Comment (v4hn) [reply core/src/stages/connect.cpp]: There is no overload for this without a parameter and `trajectory->getWayPointDurationFromStart(trajectory->getWayPointCount())`
is not any more readable or efficient than the current one.

A new `trajectory->getDuration()` function in MoveIt would be appreciated, as the class already provides a `getAverageSegmentDuration()` too...

**Comment by tylerjw** [comment]
> Did you leave out the `makeSequential` case of `Connect` on purpose @tylerjw ?

No, good catch.  Fixed.

**Comment by v4hn** [comment]
Pushed as 7ca49bc29be75618d49f06603fc5c9f59274d91a .
Thanks @tylerjw !

See #101 for the follow-up simplification.

**Pull request closed without merging**